packages/frontend/composables/README.md

```diff
@@ -0,0 +1 @@
+# @n8n/front-end-composables
```

Comment:
Could we add a short description about what this package is about? Couple sentences is enough, can be elaborated in the future more

Reply:
Added a basic description